README.md

```diff
@@ -12,6 +12,23 @@ Main motivation for Intl.DisplayNames project was to enable developers to get tr
 * Reduce translation cost for developers.
 * Consistent translation of language, region and script display name on the web.
 
+### Scope
+This proposal is not intended to provide a translation for all strings, but rather focus on particular items which are application-independent. There are two classes of strings we're considering here:
```

Comment:
I suggest we start with a positive tone:
"This proposal is intended to provide translation for strings of particular items which are application-independent, rather than translation for all kinds of strings. There are two classes of strings we're considering here:"